docs/source/api.rst

```diff
@@ -0,0 +1,60 @@
+*****************
+API Documentation
+*****************
+
+.. currentmodule:: merlin.systems
+
+
+Workflow Graph Constructors
+---------------------------
+
+.. currentmodule:: merlin.systems.dag
+
+.. autosummary::
+   :toctree: generated
+
+   Ensemble
+   Node
```

Comment:
We don't expect that users will need to interact with `Node` directly, so not sure if it should be included here or not. No harm, I suppose.